lib/rubernetes/auth/kube_config.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Rubernetes
+  module Auth
+    # This class is used to read and parse a KUBECONFIG file.
+    # It will try to read the KUBECONFIG from ENV first, otherwise it will fallback to ~/.kube/config.
+    # It extracts the kube API endpoint and authentication details.
+    class KubeConfig
+      def initialize
+        config_file = ENV.fetch('KUBECONFIG', "#{Dir.home}/.kube/config")
```

Comment:
```suggestion
        config_path = ENV.fetch('KUBECONFIG', "#{Dir.home}/.kube/config")
```

To match the same name [here](https://github.com/rubernetes/gem/pull/15/files#diff-68e3f5136ef530cd204dc36c363de553185fc9feb0b8eb5beb7750c4d038f6ddR36)